README.md

```diff
@@ -47,6 +47,41 @@ $ npm run generate
 - Suggestions Page (Suggestions.vue)
   - ~\screens\SuggestionModal.vue
 
+### Folders
+
+- Assets
+  - CSS/SCSS Files
+  - Fonts
+    - One Folder per font
+      - all font files
+      - a font style sheet
+  - Images, the ones that are unique to each page
+  - SVG files
+  - Utils(not to be confused with ~\utils), idk either its a big mystery to me.
```

Comment:
clarify? I wouldn't leave this in the final product lol

Reply:
LOL o those are for like the vuesax import tbh they should just be included into ~/utils but havent gotten around to it